modules/doc/content/newsletter/2021_01.md

```diff
@@ -27,3 +27,11 @@ Failure analysis reports were added to the SQA documentation, see [framework_far
 
 Documentation for setting up MOOSE-based applications to leverage capabilities of MooseDocs
 for SQA was added: [extensions/sqa.md].
+
+# Strumpack Support
```

Comment:
```suggestion
## Strumpack Support
```
@fdkong I would make the heading a little smaller to fit with the rest.

Reply:
Thanks!